webapp-django/crashstats/authentication/management/commands/makesuperuser.py

```diff
@@ -1,6 +1,9 @@
-from __future__ import print_function
+"""
+Makes a user into a superuser/staff. This is helpful for local development
+environment.
+"""
```

Comment:
While working on `auditgroups`, I needed `makesuperuser` to add me to the `Hackers` group. Then I looked at the code and was like, "ugh", and fixed a bunch of issues in the command code and the test.